.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
+- [ ] (Feedback from reviewers)
+
+## How To Test
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
```

Comment:
Ya, at this point the "How to Test" section is pretty generic. However this section may become relevant for certain future PRs that add a new feature such as configuration for new or existing linters.